benchmarks/apps/sphng/sphng.py

```diff
@@ -0,0 +1,223 @@
+import os.path as path
+import reframe as rfm
+import reframe.utility.sanity as sn
+from benchmarks.modules.utils import SpackTest
+
+
+class SphngBase(SpackTest):
+
+    descr = 'Base class for Sphng'
+    valid_systems = ['*']
+    valid_prog_environs = ['*']
```

Comment:
```suggestion
    valid_prog_environs = ['default']
```
Could you enforce that these only run in the default programming environment? I'm going to submit another PR to clean up our environments, they don't do anything when we are building with spack, but just to avoid possible issues in the future. I tried to run these on Cosma8 and got flooded with benchmarks running in all environments.

Reply:
done